actors/builtin/storage_power/storage_power_actor.go

```diff
@@ -353,15 +381,20 @@ func (a *StoragePowerActor) Constructor(rt Runtime) *vmr.EmptyReturn {
 func (a *StoragePowerActor) _rtAddPowerForSector(rt Runtime, minerAddr addr.Address, storageWeightDesc SectorStorageWeightDesc) {
 	var st StoragePowerActorState
 	rt.State().Transaction(&st, func() interface{} {
-		st._addClaimedPowerForSector(rt, minerAddr, storageWeightDesc)
+		if err := st._addClaimedPowerForSector(adt.AsStore(rt), minerAddr, storageWeightDesc); err != nil {
+			rt.Abort(exitcode.ErrIllegalState, "failed to add power power for sector")
+		}
 		return nil
 	})
 }
 
 func (a *StoragePowerActor) _rtDeductClaimedPowerForSectorAssert(rt Runtime, minerAddr addr.Address, storageWeightDesc SectorStorageWeightDesc) {
 	var st StoragePowerActorState
 	rt.State().Transaction(&st, func() interface{} {
-		st._deductClaimedPowerForSectorAssert(rt, minerAddr, storageWeightDesc)
+		if err := st._deductClaimedPowerForSectorAssert(adt.AsStore(rt), minerAddr, storageWeightDesc); err != nil {
+			rt.Abort(exitcode.ErrIllegalState, "failed to deducted claimed power for sector")
```

Comment:
This non-exported method should probably return the wrapped error rather than abort.

Reply:
The name of this method seems to imply it will make some assertions about the success of the operation it performs. We can always change the name, but given how it reads _now_, returning errors feels wrong and also a big ugly:
```go
func (a *StoragePowerActor) _rtDeductClaimedPowerForSectorAssert(rt Runtime, minerAddr addr.Address, storageWeightDesc SectorStorageWeightDesc) error {
	var st StoragePowerActorState
	var txErr error
	rt.State().Transaction(&st, func() interface{} {
		if err := st._deductClaimedPowerForSectorAssert(adt.AsStore(rt), minerAddr, storageWeightDesc); err != nil {
                        // unless I can just return the error directly here??
			txErr = err
			return nil
		}
		return nil
	})
	return errors.Wrap(txErr, "failed to deducted claimed power for sector")
}
```
How strongly do we feel about errors propagating up from here (also cc @whyrusleeping)

Reply:
Yeah please change the name.